apps/metadata-editor/src/app/records/search-records/search-records-list.component.html

```diff
@@ -1,9 +1,58 @@
-<md-editor-records-list
-  [title]="
-    (searchText$ | async)
-      ? ('dashboard.records.search'
-        | translate: { searchText: searchText$ | async })
-      : ('dashboard.records.all' | translate)
-  "
->
-</md-editor-records-list>
+<main class="bg-white">
+  <div class="flex flex-row items-baseline gap-[8px] px-[32px] py-[20px]">
+    <ng-container *ngIf="searchText$ | async as searchText; else allRecords">
+      <h1
+        class="text-[16px] text-main font-title font-bold"
+        translate
+        [translateParams]="{ searchText: searchText }"
+      >
+        dashboard.records.search
+      </h1>
+      <div class="text-[12px]">
+        <md-editor-records-count></md-editor-records-count>
+      </div>
+    </ng-container>
+    <ng-template #allRecords>
+      <h1 class="text-[16px] text-main font-title font-bold" translate>
+        dashboard.records.all
+      </h1>
+      <div class="text-[12px]">
```

Comment:
I don't mind for gap, padding, margin, but should text size be hardcoded in pixels this way?